dev/.rat-excludes

```diff
@@ -2,4 +2,5 @@
 .rat-excludes
 LICENSE
 NOTICE
-go.sum
\ No newline at end of file
+go.sum
+build
```

Comment:
In https://github.com/apache/iceberg/blob/master/dev/.rat-excludes#L8 we're ignoring the entire `build` folder (where the RAT results lands), so we're probably ok to do the same here